src/helpers/createTestHarness.tsx

```diff
@@ -39,4 +39,13 @@ export const createTestHarness = <TProps, TResult>(
     }
     return component
   }
+
+  // If the function name does not get used before it is returned,
+  // it seems to vanish in nodejs and does not appear in stack traces.
+  // This dummy usage works around that.
+  const _name = testHarness.name // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
[same suggestion](https://github.com/testing-library/react-hooks-testing-library/pull/528/files#r553863248)

```suggestion
  // If the function name does not get used before it is returned,
  // it's name is removed by babel-plugin-minify-dead-code-elimination
  // This dummy usage works around that.
  testHarness.name
```